src/appleseedmaya/exporters/lightexporter.cpp

```diff
@@ -47,6 +47,17 @@
 namespace asf = foundation;
 namespace asr = renderer;
 
+namespace
+{
+    void addIndirectLightControls(const MObject& node, asr::ParamArray& lightParams)
+    {
+        bool castIndirectLight = true;
+        assert(AttributeUtils::get(node, "asCastIndirectLight", castIndirectLight) &&
```

Comment:
There's a bug here. In optimized builds, `x` is not evaluated in `assert(x)`, i.e. the entire `assert()` statement is ignored.

Reply:
Thanks, fixed.